blazor/floating-action-button/positions.md

```diff
@@ -0,0 +1,145 @@
+        font-weight: normal;
+        font-style: normal;
+    }
+
+    [class^="fab-icon-"],
+    [class*=" fab-icon-"] {
+        font-family: 'fab-icons' !important;
+        speak: none;
+        font-size: 55px;
+        font-style: normal;
+        font-weight: normal;
+        font-variant: normal;
+        text-transform: none;
+        line-height: inherit;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
+    }
+
+    .fab-icon-people:before {
+        content: "\e70a";
+    }
+
+</style>
+
+```
+
+![Blazor FAB Component with different positions](./images/FabPositions.png)
+
+## Custom position
+
+You can define the custom position of the Floating Action Button by override the `top`, `left`, `right`, and `bottom` CSS properities using `CssClass`. For detailed information, refer file below.
```

Comment:
Remove "For detailed information, refer file below." here and used other places.